File: lab3/Lab3-Serious-Form/Lab3-Serious-Form/wwwroot/src/app/models/review.ts
export class Review
{
  name : string
  email : string
  message : string
  phoneNumber: string

  constructor(
    reviewName: string,
    reviewEmail: string,
    reviewMessage: string,
    reviewPhoneNumber: string)
  {
    this.name = reviewName;
    this.email = reviewEmail;
    this.message = reviewMessage;
    this.phoneNumber = reviewPhoneNumber;
  }
}

File: lab3/Lab3-Serious-Form/Lab3-Serious-Form/Entity/Review.cs
namespace Lab3_Serious_Form.Entity
{
    public class Review : PersistableEntity
    {
        public string Name { get; set; } = string.Empty;
        public string Email { get; set; } = string.Empty;
        public string Message { get; set; } = string.Empty;
        public string PhoneNumber { get; set; } = string.Empty;
    }
}